Sources/TuistKit/ContentHashing/GraphContentHasher.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import TuistCore
+
+public protocol GraphContentHashing {
+    func contentHashes(for graph: Graphing) -> Dictionary<TargetNode, Int>
```

Comment:
```suggestion
    func contentHashes(for graph: Graphable) -> Dictionary<TargetNode, Int>
```

Reply:
Reverting this change because it's big and got conflicts, @pepibumur is shipping to many things